### DISCUSSION TITLE
Allow the smelting products of raw ore to be specified as ingots from other mods

### DISCUSSION THREAD

**Discussion by Omgise** [Roadhog360/Et-Futurum-Requiem#536] [Ideas]
This can better unify the ores.

**Comment by Roadhog360** [comment]
Why can't you use CraftTweaker to do that?

**Reply by Omgise** [reply]
I don't know how to use CraftTweaker

**Reply by Roadhog360** [reply]
Learn I guess

**Reply by Omgise** [reply]
Alright, I'll look for the example documentation of CraftTweaker and study it.

**Reply by Omgise** [reply]
Before, I couldn't find its example documentation 😢

**Reply by Roadhog360** [reply]
https://docs.blamejared.com/MineTweaker/en/index

I should add this to resources in Legacy Modding

**Reply by Omgise** [reply]
Thanks！
